src/tests/auth.test.js

```diff
@@ -29,24 +29,25 @@ const user2 = {
 };
 
 describe('/api/v1/auth', () => {
-  let verifiedUser, notVerifiedUser;
+  let verifiedUser, notVerifiedUser, unverifiedUser;
 
-  before(async () => {
-    await TestHelper.destroyModel('User');
-  });
-
-  describe('POST /login', () => {
-    before(async () => {
-      verifiedUser = await TestHelper.createUser({
-        ...user, is_verified: true
-      });
+  beforeEach(async () => {
+    verifiedUser = await TestHelper.createUser({
+      ...user, is_verified: true
+    });
 
-      notVerifiedUser = await TestHelper.createUser({
-        ...user,
-        email: 'user2@gmail.com',
-      });
+    notVerifiedUser = await TestHelper.createUser({
+      ...user,
+      email: 'user2@gmail.com',
     });
+  });
 
+  afterEach((done) => {
+    TestHelper.destroyModel('User');
+    done();
+  });
```

Comment:
Why did you move this test from `before` to `afterEach`?

Reply:
This was to prevent the test data from affecting other tests that would run after it

Reply:
I did it while I was fixing an issue with the tests. Amos didn't properly use hooks to seed his test data, so some of his tests were failing when tests that ran before it altered the User table in the test database.

Reply:
You cannot change things to make your own test pass instead fix the test itself, these are kind of changes that messes up with our application.

Reply:
Alright